.travis.yml

```diff
@@ -7,6 +13,7 @@ script:
 - npm run check-coverage
 - npm run upload-coverage-coveralls
 - npm run upload-coverage-codacy
-notifications:
-  slack:
-    secure: cKmjGeAX1JWRw+YRUsAn+IxUTenokF35bP1Uq1revB2AJ1e9oE9fyPaLLdTHwa6tRsqhraC3OM9VRkGkkK2xp8lDifLe3VPsKx0C0EFaFaa3Y5XGM4dtL7YBaIQV45QpXwsO/kMa6wKOyL1wlN9Eo8+9rHpu9mulwRmgUYVmICWvXxD8v1oleXwbNeUfXatVa1x6qt8anF2BEAPvjJ3oSVd87EIBHByYIzkMWgcB0FsWVp19NMAvDsa/DjTs0qb0gi5W4Q/TOjtse5Ycv+esiUo2OsUww2QDaGoATjI39W+1H3IGFSsZWeuSGqZmZnAGbUSfz5fsrbTPEHYpiRGBiZ1YktkuXMWzuMRgD5UklZzPY5LxW7a8BwMo1pERSpT7mWEw7JSenL96d+bPONWjQtvx8imS0VWLhMA9t13AtzPHWRC4VUc0fJyAxUWr+T+8tBMnHocSOXXfdbFrfPfYCDmRE8gr3939+yd9xEGYN26q/4l4aLAufZBQJcFhrSaHVsilTmamIXF/HwWwkmaMQUZtKDIPr6OncEe6vrhwhObceLsLNLmDxobNCyEVBVayaFzOMgDWiYIv1x0nd2pK42L6ZQfXJwe1Ex+kmVm2m39HKcC4vwN92+WDn1noL2G6H0spKXrnHRMcI0Rx2C2F8AJEAuICuyK1YV6eNFAoRig=
+- docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+- chmod +x ./scripts/*
+- ./scripts/docker_ci.sh
```

Comment:
Shouldn't need to chmod here - would be better the chmod the script files and commit. Git preserves the x flag but not permissions.